packages/cli/src/commands/repo/build.ts

```diff
@@ -153,9 +153,7 @@ export async function command(opts: OptionValues, cmd: Command): Promise<void> {
           return;
         }
 
-        const configPaths = opts.config?.length
-          ? opts.config
-          : buildOptions.config ?? [];
+        const configPaths = buildOptions.config ?? opts.config ?? [];
```

Comment:
Is buildOptions.config undefined if there are no config overrides? Just want to make sure we don't end up always using that because [] is truthy.